src/components/Select/Select.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+export interface Options {
+  label: string
+  value: string
+}
+
+export interface Props {
+  className?: string
+  value: string
+  name: string
+  required?: boolean
+  disabled?: boolean
+  error?: boolean
+  width?: number | string
+  options: Options[]
+  blankOption?: boolean
+  placeholder?: string
+  onChange?: (name: string, value: string) => void
```

Comment:
Please add `aria-label` property for a11y.
And check Accessibility panel in storybook !

![image](https://user-images.githubusercontent.com/1614892/66287208-c2977d80-e90f-11e9-8a25-606835ef0ad7.png)

Reply:
Fixed here.
8a7a0ab

I'll be careful to check the accessible panel of the storybook!